content/3.community/3.contribute.md

```diff
@@ -30,64 +32,79 @@ Review existing <a href="https://github.com/archway-network/archway-docs/issues"
 
 We welcome contributions to the docs, tutorials and guides.
 
-Our technical content follows the <a href="https://developers.google.com/style" target="_blank" >Google developer documentation style guide</a>. Highlights to help you get started:
+Our technical content follows the [Google developer documentation style guide](https://developers.google.com/style). Highlights to help you get started:
 
-- <a href="https://developers.google.com/style/highlights" target="_blank" >Highlights</a>
-- <a href="https://developers.google.com/style/word-list" target="_blank" >Word list</a>
-- <a href="https://developers.google.com/style/tone" target="_blank" >Style and tone</a>
-- <a href="https://developers.google.com/style/translation" target="_blank" >Writing for a global audience</a>
-- <a href="https://developers.google.com/style/cross-references" target="_blank" >Cross-references</a>
-- <a href="https://developers.google.com/style/tense" target="_blank" >Present tense</a>
+- [Highlights](https://developers.google.com/style/highlights)
+- [Word list](https://developers.google.com/style/word-list)
+- [Style and tone](https://developers.google.com/style/tone)
+- [Writing for a global audience](https://developers.google.com/style/translation)
+- [Cross-references](https://developers.google.com/style/cross-references)
+- [Present tense](https://developers.google.com/style/tense)
```

Comment:
Based on sugestions it seems we should be using the other method which allows us to set a _blank target so external links open a new window/tab.
```suggestion
- [Highlights](https://developers.google.com/style/highlights)
- [Word list](https://developers.google.com/style/word-list)
- [Style and tone](https://developers.google.com/style/tone)
- [Writing for a global audience](https://developers.google.com/style/translation)
- [Cross-references](https://developers.google.com/style/cross-references)
- [Present tense](https://developers.google.com/style/tense)
```